src/main/java/org/springframework/data/elasticsearch/client/reactive/ReactiveRestClients.java

```diff
@@ -41,4 +42,18 @@ public static ReactiveElasticsearchClient create(ClientConfiguration clientConfi
 
 		return DefaultReactiveElasticsearchClient.create(clientConfiguration);
 	}
+
+	/**
+	 * Start here to create a new client tailored to your needs.
+	 *
```

Comment:
add javadoc for the parameters, mentioning that they must not be null (please fix this in the other method as well)

Reply:
Added the javadoc. Please let me know if it's enough.